### ISSUE TITLE
Warning youre all wasting your time botting 

### ISSUE THREAD

**Issue by JewBagel** [finndev/PokeBuddy#1185]
Everyone botting youre wasting your time no matter what you will be banned i promise you it may not be today it may not be tomorrow but you will be banned. just warning everyone its not worth it anymore my account was bot free for about a week and a half 2 weeks still got banned.. they're even banning legit accounts and not thinking twice about it feel bad for those people since theres really nothing they'll help you with because they could care less about there user base all they see is money cha ching...

**Comment by Kokainium** [comment]
Its true happend to med yesterday

**Comment by Kokainium** [comment]
Me*

**Comment by JewBagel** [comment]
yeah man sucks if i was a game company like this i would look into why people are even botting in first place and realize the makers ruined this game within a week of releasing it but nope lets just ban people because thats how broken games work rebelling means nothing to them

**Comment by Kokainium** [comment]
The reason why i was using bot was the locations of the pokemons :/

**Comment by JewBagel** [comment]
im sure a lot of people were but they dont care that pretty much everything is in major cities.

**Comment by Kokainium** [comment]
Indeed now i get the outofrange servers is down Thing:s

**Comment by JewBagel** [comment]
outofrange is that something new i dont get that i get the cant find account information or whatever

**Comment by Kokainium** [comment]
Its a softban

**Comment by yoyog** [comment]
If it says can't get game data your erased

**Comment by Kokainium** [comment]
Then u are perm banned

**Comment by JewBagel** [comment]
some are saying it can go away after a few days so i think its them deciding weather to actually permaban you

**Comment by Demolition1986** [comment]
@Kokainium U dont need a bot if u dont have poke in ur location, Prefer to use spoof in that way not bot oxaoxoa.

**Comment by ShupengHu** [comment]
After my main account (lvl 40, full pokedex, 6 max cp dragonites) has been perm banned, now I dont care any bans because it is free to create a new account and boost it with bot easily. I dont spend any time or money on it. Instead, I just need an account with good levels to see the future updates.  The return to this game depends on the future updates.

**Comment by JewBagel** [comment]
idk how you think that way why does it matter if you botted another account up all the way youll still get banned how is that not getting thru peoples heads also when the bots all get patched and all your accounts get banned wtf are you going to do just look back at this like wtf just wasted mad time doing that for nothing

**Comment by Demolition1986** [comment]
@JewBagel What a fuck are u doing here in a forum of bots, get lost dumbass.. If u dont believe in hacks or cheats, is not ur place to be here.. Get lost.

**Comment by hammerhead13** [comment]
Going on 5 Days since I first got the "Failed to get Game data from Server" and was able to play.

**Comment by Demolition1986** [comment]
@hammerhead13 Are u able to play right now after failed to get game data?

**Comment by HenghaoZhou** [comment]
@hammerhead13 Is that true? you got unbanned after 5 days? Have you submit an appeal to Niantic?

**Comment by hammerhead13** [comment]
No. Im still not able to Play. Its been 5 days since i was able to play. I still get that Error

**Comment by JewBagel** [comment]
Demolition go fuck yourself dude first off this isn't a forum you dumbfuck second of all wtf are you saying I'm telling you your going to get banned so you get butthurt well time to face the reality kiddo... you ain't nothing special dude I can tell you think you're but nope you'll be banned with the rest of us soon.

**Comment by RuperSama** [comment]
the error "Failed to get Game data from Server" is a permanent ban, your data is delete for the server and you cant login ever again.

Sorry @hammerhead13 but you make some mistake and the system ban you

**Comment by Abismo** [comment]
@RuperSama Yes, they delete your data, but not immediately. If you just got banned there is still a chance for you.
This is an account banned 5 days ago:
![asd](https://cloud.githubusercontent.com/assets/2502938/17713690/6929af52-63fc-11e6-8cdf-370f2fadd30c.png)

And this is an account banned yesterday:
![asdd](https://cloud.githubusercontent.com/assets/2502938/17713698/735772fc-63fc-11e6-994d-69f23cfc1246.png)

As you can see the data is still there. And 2 of my accounts have been unbanned so it's not always permanent.

**Comment by RuperSama** [comment]
@Abismo  how you get the unbanned? i send ticket in the faq page and i get all the time a bot with the same text. never get human answer (my accout have 6 days ban)

**Comment by Rab89** [comment]
they dont delete your data they sieze your account before deciding your fate, if thatmeans stopping your account from using the servers so be it.

ban wave number 2 survived :white_check_mark:

**Comment by JewBagel** [comment]
Abismo what are you using to check the account and how long have you had to wait for an account to start working again?

**Comment by naor173** [comment]
@JewBagel yo jew you are right bro i agree and i also made a status like this aswell. its a matter of time when  they will banned you.. i dont think its worth it anymore they are really into finish the third party ability to shut them all down my main still survive.. i fiinsh bot there when they started the soft ban waves

**Comment by JewBagel** [comment]
Exactly and it's like is it so hard to believe for some of you people you're not invincible we will all be banned it's just a matter of time.

**Comment by Abismo** [comment]
@RuperSama I didn't do anything. My account has been unbanned like 12 hours after the ban. Niantic won't send you any email. Despite having gained so much money with PoGo, they don't have anyone working on support to players. No one reads your emails, they even don't pick up the phone if you call them. So don't wait for that email. If your account has been banned 6 days ago it's permanent.

@JewBagel I used ALL G (you can get it in the PlayStore) in the screenshots. It took me ~12 hours to get my accounts unbanned.

I don't know if they will ban all bots (I don't think so but whatever). I'm pretty sure that at least half of the players have used bots or spoofing in this game and the problem is that the game itself is unfair. If you don't live in a big city (and I mean a very big city) you have 2 options: you spoof or you quit the game. I keep botting because why not, I don't expect to keep these accounts after their "ban waves" but if I do, well, maybe I'll go outside and play with them later...

**Comment by alanbrown123** [comment]
The one that went to 0 still banned i assume

**Comment by Abismo** [comment]
Both of them are still banned. But the first one is permanent for sure. The second one maybe not...

**Comment by JewBagel** [comment]
well my account is 16 hours in no unban yet hoping it will actually happen and im not completey fucked anyone else get unbanned?

**Comment by JewBagel** [comment]
my account is level 0 on that app so im guessing theres no hope for me at this point

**Comment by Abismo** [comment]
Aaaand... Just if anyone was wondering, that lvl 24 account got unbanned after 1 day.
![captura de pantalla 73](https://cloud.githubusercontent.com/assets/2502938/17751929/ba92621c-64c9-11e6-8816-14bafa39e5a6.png)

**Comment by alanbrown123** [comment]
Bluestacks better than nox?

**Comment by Abismo** [comment]
I never used Nox so I don't know but I think they can detect it easily looking at device properties.
